### PR TITLE
Add rules about features

### PR DESCRIPTION
**Pull request by DonRai**
Add rules about how working with features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,20 @@ test('getCartItemOptions', () => {
 * `isSomethingActive` - checking the state of the local item.
 * `hasSomething` - checking the existing of the local item.
 
+## 10. Работа с фичами
+#### 10.1. Новые фичи закрываются флагом.
+#### 10.2. Новые фичи кладутся в папку features (она лежит на одном урвоне с папкой modules). Внутри этой папки создается папка для фичи.
```
Comment (binary-data) [inline README.md]: Опечатк в слове "урвоне"

Comment (DonRai) [reply README.md]: thanks, fixed

**Comment by pbelyaev** [comment]
What if feature impact some general component (e.g. header)? Copy-paste header component into features folder?

**Comment by Sustrett** [comment]
I suggest to add following: **Feature under flag, until it is on 100%, should be handled and supported by team who created this flag.** This means that bugs, wrong behavior which will occur under the flag will be handled by team which is fully aware of feature that was implemented.

**Pull request merged**